### PR TITLE
Health food movement

### LINKED ISSUE #28: Goal 1: Health food falls from the top
I also see a picture of friends behind the Empty Calorie Overlord. 
They release healthy foods.

### PR DESCRIPTION
**Pull request by chompasina**
Adds health food movements and tests. 
Closes #28 and #45

### CODE REVIEW

File: test/health-food-test.js [hunk 1]
```diff
@@ -38,52 +38,67 @@ describe('healthFood', function(){
...
-    assert.equal(context.fillRect.calls[0][1], healthFood.y);
-    assert.equal(context.fillRect.calls[0][2], healthFood.width);
-    assert.equal(context.fillRect.calls[0][3], healthFood.height);
-  });
-
-  it('should set the correct fillstyle on draw', function(){
-    assert.equal(context.fillStyle, "green");
-  });
+// describe('draw', function(){
+//   var context = stub().of("fillRect");
```
Comment (karinamzalez) [inline test/health-food-test.js]: we should go ahead and add these tests!

**Review by karinamzalez — COMMENTED** [review]
fun pairing with you both on this! :) 🎉

**Comment by karinamzalez** [comment]
fun pairing with you both on this! :) 🎉

File: README.md [hunk 2]
```diff
@@ -1,6 +1,17 @@
...
-* We started with creating the canvas and tray. 
+### Game Name
+
+Empty Calorie Adventure
+
+### Description
+Gametime is a Module 4 project at Turing. We are building a game with a healthy eating theme. This game will eventually be included in our Pahlka Posse's healthy eating app for children.
+
+
+### How to Run
```
Comment (chompasina) [inline README.md:14]: Thanks for updating the Readme, @deborahleehamel !

**Comment by chompasina** [comment]
Loved the code that @deborahleehamel wrote while driving on my computer. We rocked this iteration!

**Pull request merged**